### PR TITLE
Upgrade TailwindCSS

### PR DESCRIPTION
**Pull request by paddockDev**
I tried my hand at upgrading TailwindCSS #182 and hope I didn't miss anything.

### CODE REVIEW

**Comment by christophrumpel** [comment]
Hey @paddockDev, thanks for the PR and helping out.

I noticed a few things not working yet:

* submit modal is blurred
* icons wrong color or gone


CleanShot 2022-02-04 at 14 41 50@2x


CleanShot 2022-02-04 at 14 41 26@2x
CleanShot 2022-02-04 at 14 41 40@2x

**Comment by christophrumpel** [comment]
Thanks, the icon seems to be ok now. But I couldn't find out why the submit modal is still completely blurred. The blur should only be on the background.

File: resources/views/pages/partials/submit-modal.blade.php [hunk 1]
```diff
@@ -15,13 +15,13 @@ class="fixed z-30 inset-0 overflow-y-auto"
...
     >
-        <div class="fixed inset-0 backdrop-filter backdrop-blur-lg backdrop-brightness-75 transition" aria-hidden="true"></div>
+        <div class="fixed inset-0 backdrop-blur-lg backdrop-brightness-75 transition" aria-hidden="true"></div>
 
         <!-- This element is to trick the browser into centering the modal contents. -->
         <span class="hidden sm:inline-block sm:align-middle sm:h-screen" aria-hidden="true">&#8203;</span>
 
         <!-- Modal Content -->
-        <div class="inline-block align-bottom bg-white rounded-lg p-6 sm:p-12 text-left overflow-hidden shadow-xl transform transition-all sm:my-8 sm:align-middle sm:max-w-lg sm:w-full"
+        <div class="inline-block align-bottom bg-white rounded-lg p-6 sm:p-12 text-left overflow-hidden shadow-xl transition-all sm:my-8 sm:align-middle sm:max-w-lg sm:w-full"
```
Comment (spaantje) [inline resources/views/pages/partials/submit-modal.blade.php:24]: If you add the `relative` class to this div the modal should be above the backdrop.

Comment (christophrumpel) [reply resources/views/pages/partials/submit-modal.blade.php]: That worked indeed :-) Thanks a lot.

**Comment by christophrumpel** [comment]
Thanks for the PR @paddockDev, I think we got it now 👍

**Comment by mathiasonea** [comment]
closed #187

**Pull request merged**